src/dpo/test/aes.ok

```diff
@@ -24,42 +24,42 @@ Importing netlist into detailed improver.
-[INFO DPO-0335] Generator displacement, Cumulative attempts 377660, swaps 223505, moves 149717 since last reset.
-[INFO DPO-0333] End of pass, Objective hpwl, Initial cost 7.305532e+08, Scratch cost 7.269701e+08, Incremental cost 7.269701e+08, Mismatch? N
-[INFO DPO-0338] End of pass, Total cost is 7.269701e+08.
-[INFO DPO-0327] Pass   1 of random improver; improvement in cost is 0.49 percent.
-[INFO DPO-0328] End of random improver; improvement is 0.490471 percent.
+[INFO DPO-0332] End of pass, Generator displacement called 1888300 times.
+[INFO DPO-0335] Generator displacement, Cumulative attempts 1888300, swaps 1022686, moves 795675 since last reset.
+[INFO DPO-0333] End of pass, Objective hpwl, Initial cost 7.068099e+08, Scratch cost 7.065984e+08, Incremental cost 7.065984e+08, Mismatch? N
+[INFO DPO-0338] End of pass, Total cost is 7.065984e+08.
+[INFO DPO-0327] Pass   1 of random improver; improvement in cost is 0.03 percent.
+[INFO DPO-0328] End of random improver; improvement is 0.029929 percent.
+[INFO DPO-0303] Running algorithm for global swaps.
+[INFO DPO-0306] Pass   1 of global swaps; hpwl is 7.087383e+08.
+[INFO DPO-0307] End of global swaps; objective is 7.087383e+08, improvement is 0.00 percent.
+[INFO DPO-0303] Running algorithm for reordering.
+[INFO DPO-0304] Pass   1 of reordering; objective is 7.086082e+08.
+[INFO DPO-0305] End of reordering; objective is 7.086082e+08, improvement is 0.02 percent.
+[INFO DPO-0303] Running algorithm for vertical swaps.
+[INFO DPO-0308] Pass   1 of vertical swaps; hpwl is 7.085952e+08.
+[INFO DPO-0309] End of vertical swaps; objective is 7.085952e+08, improvement is 0.00 percent.
+[INFO DPO-0303] Running algorithm for independent set matching.
+[INFO DPO-0300] Set matching objective is wirelength.
+[INFO DPO-0301] Pass   1 of matching; objective is 7.085952e+08.
+[INFO DPO-0302] End of matching; objective is 7.085858e+08, improvement is 0.00 percent.
 [INFO DPO-0380] Cell flipping.
-[INFO DPO-0382] Changed 9488 cell orientations for row compatibility.
-[INFO DPO-0383] Performed 6147 cell flips.
-[INFO DPO-0384] End of flipping; objective is 7.245356e+08, improvement is 0.63 percent.
+[INFO DPO-0382] Changed 9460 cell orientations for row compatibility.
+[INFO DPO-0383] Performed 5689 cell flips.
+[INFO DPO-0384] End of flipping; objective is 7.058162e+08, improvement is 0.39 percent.
 [INFO DPO-0313] Found 0 cells in wrong regions.
 [INFO DPO-0315] Found 0 row alignment problems.
 [INFO DPO-0314] Found 0 site alignment problems.
 [INFO DPO-0311] Found 0 overlaps between adjacent cells.
 [INFO DPO-0312] Found 0 edge spacing violations and 0 padding violations.
 Detailed Improvement Results
 ------------------------------------------
-Original HPWL           372201.2 u
-Final HPWL              360418.2 u
-Delta HPWL                  -3.2 %
+Original HPWL           351168.7 u
+Final HPWL              350977.5 u
+Delta HPWL                  -0.1 %
```

Comment:
Why is the Original HPWL reduced?  I would expect on the Final & Delta to change.

Reply:
I change the also the design def so that will be the original for the regression tests later on.
You can see the comparison between the two set of commands in the image the I upload above.

Reply:
What was the motivation to change the input def?  How was it modified?

Reply:
Maybe I misunderstood the flow.
As I understand it, is that every test should be identical in every regression run.
So if the new set of commands approved it need to change also the original def in dpo test since when there will another test running there will be def between the new def to the original because the original will not got the def with the new improve placement commands.

Anyway I can change it again, I just need to understand the correct flow

Reply:
All the unit tests are standalone and don't depend on other tests.  Generally you would update the *ok files not the input files.